src/categorical_algebra/CSets.jl

```diff
@@ -102,6 +102,9 @@ For morphisms, the result is a `FinFunction`; for attributes, a
 @inline SetFunction(X::StructACSet{S}, name::Symbol) where {S} =
   set_function(X, Val{S}, Val{name})
 
+@inline SetFunction(X::DynamicACSet, name::Symbol)  =
```

Comment:
This one shouldn't be inlined. The inlining is only required to propagate constants through `Val` types.